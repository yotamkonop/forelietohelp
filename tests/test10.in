add_song 1040904 378
add_playlist 776276
get_plays 1040904
add_to_playlist 776276 1040904
add_song 325188 538
get_num_songs 776276
add_song 925774 868
add_to_playlist 776276 925774
get_by_plays 776276 531
add_song 335246 507
get_num_songs 776276
add_song 565247 636
add_song 122918 354
add_playlist 270888
get_plays 122918
delete_song 325188
add_playlist 938055
delete_playlist 270888
delete_song 335246
add_playlist 807178
add_to_playlist 776276 122918
add_playlist 403781
get_num_songs 807178
add_to_playlist 776276 565247
get_num_songs 776276
add_to_playlist 807178 1040904
get_num_songs 403781
add_to_playlist 807178 335246
add_playlist 435628
add_to_playlist 938055 1040904
add_song 127770 250
get_num_songs 807178
add_playlist 649706
add_song 437591 261
get_plays 122918
add_to_playlist 776276 127770
get_by_plays 938055 354
delete_song 437591
get_num_songs 938055
add_to_playlist 807178 565247
get_by_plays 403781 135
add_to_playlist 807178 122918
add_playlist 142858
add_to_playlist 142858 122918
get_num_songs 649706
add_song 865101 389
get_num_songs 807178
get_plays 127770
add_to_playlist 938055 122918
add_song 479901 663
add_to_playlist 403781 127770
delete_playlist 435628
get_plays 565247
add_to_playlist 403781 865101
get_by_plays 270888 433
get_num_songs 776276
add_to_playlist 776276 479901
add_song 283021 813
add_song 242867 205
get_num_songs 649706
add_song 934670 512
get_by_plays 807178 95
get_plays 122918
delete_song 242867
add_song 764096 751
add_to_playlist 435628 1040904
add_song 439328 667
add_song 291191 892
add_to_playlist 403781 934670
delete_song 283021
add_to_playlist 649706 865101
get_num_songs 403781
get_by_plays 938055 258
get_num_songs 403781
add_playlist 332784
add_to_playlist 807178 127770
add_to_playlist 776276 764096
add_to_playlist 807178 934670
get_plays 127770
add_to_playlist 807178 865101
add_song 4652 307
get_plays 4652
add_playlist 425711
add_to_playlist 403781 1040904
add_playlist 817919
get_plays 291191
add_to_playlist 938055 439328
get_num_songs 142858
get_num_songs 142858
add_song 133902 347
add_to_playlist 142858 925774
get_by_plays 776276 336
add_song 230803 312
remove_from_playlist 649706 127770
get_num_songs 649706
get_plays 565247
add_to_playlist 649706 127770
get_num_songs 807178
add_song 712734 821
add_to_playlist 332784 230803
add_song 244597 114
get_num_songs 817919
add_to_playlist 332784 122918
add_to_playlist 332784 865101
get_by_plays 649706 761
add_song 847372 621
get_by_plays 425711 593
get_num_songs 649706
add_song 547131 180
get_num_songs 817919
get_plays 847372
get_num_songs 649706
get_num_songs 142858
get_num_songs 938055
add_to_playlist 142858 4652
get_by_plays 403781 17
add_song 788082 890
unite_playlists 776276 142858
get_plays 847372
add_to_playlist 817919 4652
remove_from_playlist 649706 127770
add_to_playlist 807178 230803
add_playlist 22589
add_to_playlist 403781 439328
get_num_songs 817919
add_to_playlist 403781 547131
add_playlist 979213
get_plays 788082
get_plays 847372
add_song 1001018 193
get_by_plays 807178 78
get_num_songs 332784
add_song 1017026 123
add_song 188472 807
get_plays 230803
add_song 753089 373
add_song 263430 342
unite_playlists 807178 22589
add_song 557233 281
add_to_playlist 270888 934670
add_to_playlist 807178 1017026
add_to_playlist 270888 283021
add_to_playlist 425711 244597
add_song 708303 695
get_plays 547131
add_song 569973 478
add_to_playlist 776276 557233
get_num_songs 403781
add_to_playlist 142858 263430
add_song 328565 79
add_to_playlist 22589 934670
add_to_playlist 776276 712734
get_num_songs 807178
get_num_songs 142858
get_num_songs 425711
add_to_playlist 938055 133902
add_song 351679 884
add_to_playlist 425711 291191
delete_song 263430
get_num_songs 425711
add_song 102289 607
add_to_playlist 817919 934670
get_num_songs 776276
get_by_plays 817919 316
get_by_plays 425711 310
add_playlist 148657
add_to_playlist 979213 547131
get_num_songs 938055
add_to_playlist 148657 1017026
add_playlist 903141
add_to_playlist 979213 1040904
get_plays 865101
add_song 827342 414
get_num_songs 649706
add_playlist 880731
get_plays 328565
get_by_plays 649706 345
add_to_playlist 649706 764096
add_to_playlist 148657 133902
add_song 231758 773
delete_song 569973
add_song 465512 418
get_plays 925774
add_to_playlist 807178 479901
get_num_songs 817919
add_song 376860 558
add_to_playlist 22589 865101
add_to_playlist 332784 188472
add_to_playlist 807178 547131
add_to_playlist 807178 102289
get_by_plays 22589 399
delete_song 753089
get_plays 934670
add_to_playlist 817919 127770
add_song 841059 785
add_to_playlist 425711 708303
add_song 139113 473
remove_from_playlist 807178 1017026
add_to_playlist 148657 139113
get_by_plays 776276 578
remove_from_playlist 403781 127770
get_plays 127770
get_plays 841059
get_plays 230803
add_to_playlist 649706 925774
get_num_songs 807178
get_by_plays 979213 205
add_playlist 646343
add_to_playlist 807178 244597
add_playlist 262218
get_plays 231758
add_song 907931 324
remove_from_playlist 425711 325188
get_plays 788082
get_plays 439328
get_num_songs 403781
get_plays 127770
add_song 191970 603
get_num_songs 148657
add_to_playlist 776276 841059
get_plays 244597
get_plays 841059
add_to_playlist 979213 376860
delete_song 231758
get_by_plays 817919 703
unite_playlists 807178 262218
add_to_playlist 776276 325188
get_plays 191970
get_plays 708303
add_to_playlist 425711 127770
add_to_playlist 979213 557233
add_to_playlist 817919 565247
get_by_plays 776276 826
add_to_playlist 270888 244597
add_playlist 445019
get_plays 1017026
delete_song 865101
add_to_playlist 445019 230803
get_num_songs 445019
delete_song 1001018
add_playlist 656316
add_song 1032726 552
add_to_playlist 332784 242867
remove_from_playlist 435628 325188
delete_song 465512
add_song 391502 416
get_by_plays 22589 373
add_to_playlist 445019 4652
get_num_songs 646343
add_song 376073 343
add_to_playlist 649706 934670