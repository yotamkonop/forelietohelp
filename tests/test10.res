add_song: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 378
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 1
add_song: SUCCESS
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 925774
add_song: SUCCESS
get_num_songs: SUCCESS, 2
add_song: SUCCESS
add_song: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 354
delete_song: SUCCESS
add_playlist: SUCCESS
delete_playlist: SUCCESS
delete_song: SUCCESS
add_playlist: SUCCESS
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 4
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_to_playlist: FAILURE
add_playlist: SUCCESS
add to playlist: should be:2  7 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 1
add_playlist: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 354
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 1040904
delete_song: SUCCESS
get_num_songs: SUCCESS, 1
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: FAILURE
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
add_playlist: SUCCESS
add to playlist: should be:2  7 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_song: SUCCESS
get_num_songs: SUCCESS, 3
get_plays: SUCCESS, 250
add to playlist: should be:3  9 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
delete_playlist: SUCCESS
get_plays: SUCCESS, 636
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: FAILURE
get_num_songs: SUCCESS, 5
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 0
add_song: SUCCESS
get_by_plays: SUCCESS, 122918
get_plays: SUCCESS, 354
delete_song: SUCCESS
add_song: SUCCESS
add_to_playlist: FAILURE
add_song: SUCCESS
add_song: SUCCESS
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
delete_song: SUCCESS
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 3
get_by_plays: SUCCESS, 122918
get_num_songs: SUCCESS, 3
add_playlist: SUCCESS
add to playlist: should be:2  7 - 3 / 2
add_to_playlist: SUCCESS
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
get_plays: SUCCESS, 250
add to playlist: should be:2  7 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 307
add_playlist: SUCCESS
add to playlist: should be:3  9 - 3 / 2
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 892
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 1
get_num_songs: SUCCESS, 1
add_song: SUCCESS
add to playlist: should be:1  5 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 122918
add_song: SUCCESS
remove_from_playlist: FAILURE
get_num_songs: SUCCESS, 1
get_plays: SUCCESS, 636
add to playlist: should be:3  9 - 3 / 2
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 6
add_song: SUCCESS
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 0
add to playlist: should be:4  11 - 3 / 2
add_to_playlist: SUCCESS
add to playlist: should be:3  9 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: FAILURE
add_song: SUCCESS
get_by_plays: FAILURE
get_num_songs: SUCCESS, 2
add_song: SUCCESS
get_num_songs: SUCCESS, 0
get_plays: SUCCESS, 621
get_num_songs: SUCCESS, 2
get_num_songs: SUCCESS, 2
get_num_songs: SUCCESS, 3
add to playlist: should be:0  3 - 3 / 2
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 127770
add_song: SUCCESS
here 1
