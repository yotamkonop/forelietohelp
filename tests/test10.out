add_song: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 378
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 1
add_song: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 925774
add_song: SUCCESS
get_num_songs: SUCCESS, 2
add_song: SUCCESS
add_song: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 354
delete_song: SUCCESS
add_playlist: SUCCESS
delete_playlist: SUCCESS
delete_song: SUCCESS
add_playlist: SUCCESS
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 4
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_to_playlist: FAILURE
add_playlist: SUCCESS
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 1
add_playlist: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 354
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 1040904
delete_song: SUCCESS
get_num_songs: SUCCESS, 1
add_to_playlist: SUCCESS
get_by_plays: FAILURE
add_to_playlist: SUCCESS
add_playlist: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_song: SUCCESS
get_num_songs: SUCCESS, 3
get_plays: SUCCESS, 250
add_to_playlist: SUCCESS
add_song: SUCCESS
add_to_playlist: SUCCESS
delete_playlist: SUCCESS
get_plays: SUCCESS, 636
add_to_playlist: SUCCESS
get_by_plays: FAILURE
get_num_songs: SUCCESS, 5
add_to_playlist: SUCCESS
add_song: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 0
add_song: SUCCESS
get_by_plays: SUCCESS, 122918
get_plays: SUCCESS, 354
delete_song: SUCCESS
add_song: SUCCESS
add_to_playlist: FAILURE
add_song: SUCCESS
add_song: SUCCESS
add_to_playlist: SUCCESS
delete_song: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 3
get_by_plays: SUCCESS, 122918
get_num_songs: SUCCESS, 3
add_playlist: SUCCESS
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
get_plays: SUCCESS, 250
add_to_playlist: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 307
add_playlist: SUCCESS
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 892
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 1
get_num_songs: SUCCESS, 1
add_song: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 122918
add_song: SUCCESS
remove_from_playlist: FAILURE
get_num_songs: SUCCESS, 1
get_plays: SUCCESS, 636
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 6
add_song: SUCCESS
add_to_playlist: SUCCESS
add_song: SUCCESS
get_num_songs: SUCCESS, 0
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: FAILURE
add_song: SUCCESS
get_by_plays: FAILURE
get_num_songs: SUCCESS, 2
add_song: SUCCESS
get_num_songs: SUCCESS, 0
get_plays: SUCCESS, 621
get_num_songs: SUCCESS, 2
get_num_songs: SUCCESS, 2
get_num_songs: SUCCESS, 3
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 127770
add_song: SUCCESS
unite_playlists: SUCCESS
get_plays: SUCCESS, 621
add_to_playlist: SUCCESS
remove_from_playlist: SUCCESS
add_to_playlist: SUCCESS
add_playlist: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 1
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 890
get_plays: SUCCESS, 621
add_song: SUCCESS
get_by_plays: SUCCESS, 127770
get_num_songs: SUCCESS, 3
add_song: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 312
add_song: SUCCESS
add_song: SUCCESS
unite_playlists: SUCCESS
add_song: SUCCESS
add_to_playlist: FAILURE
add_to_playlist: SUCCESS
add_to_playlist: FAILURE
add_to_playlist: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 180
add_song: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 6
add_to_playlist: FAILURE
add_song: SUCCESS
add_to_playlist: FAILURE
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 8
get_num_songs: FAILURE
get_num_songs: SUCCESS, 1
add_to_playlist: SUCCESS
add_song: SUCCESS
add_to_playlist: SUCCESS
delete_song: SUCCESS
get_num_songs: SUCCESS, 2
add_song: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 10
get_by_plays: SUCCESS, 934670
get_by_plays: SUCCESS, 291191
add_playlist: SUCCESS
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 4
add_to_playlist: SUCCESS
add_playlist: SUCCESS
add_to_playlist: SUCCESS
get_plays: SUCCESS, 389
add_song: SUCCESS
get_num_songs: SUCCESS, 1
add_playlist: SUCCESS
get_plays: SUCCESS, 79
get_by_plays: SUCCESS, 865101
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
add_song: SUCCESS
delete_song: SUCCESS
add_song: SUCCESS
get_plays: SUCCESS, 868
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 2
add_song: SUCCESS
add_to_playlist: FAILURE
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: FAILURE
delete_song: SUCCESS
get_plays: SUCCESS, 512
add_to_playlist: SUCCESS
add_song: SUCCESS
add_to_playlist: SUCCESS
add_song: SUCCESS
remove_from_playlist: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 565247
remove_from_playlist: SUCCESS
get_plays: SUCCESS, 250
get_plays: SUCCESS, 785
get_plays: SUCCESS, 312
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 10
get_by_plays: SUCCESS, 1040904
add_playlist: SUCCESS
add_to_playlist: SUCCESS
add_playlist: SUCCESS
get_plays: SUCCESS, 773
add_song: SUCCESS
remove_from_playlist: FAILURE
get_plays: SUCCESS, 890
get_plays: SUCCESS, 667
get_num_songs: SUCCESS, 5
get_plays: SUCCESS, 250
add_song: SUCCESS
get_num_songs: SUCCESS, 3
add_to_playlist: SUCCESS
get_plays: SUCCESS, 114
get_plays: SUCCESS, 785
add_to_playlist: SUCCESS
delete_song: SUCCESS
get_by_plays: FAILURE
unite_playlists: SUCCESS
add_to_playlist: FAILURE
get_plays: SUCCESS, 603
get_plays: SUCCESS, 695
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
add_to_playlist: SUCCESS
get_by_plays: SUCCESS, 925774
add_to_playlist: FAILURE
add_playlist: SUCCESS
get_plays: SUCCESS, 123
delete_song: FAILURE
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 1
delete_song: SUCCESS
add_playlist: SUCCESS
add_song: SUCCESS
add_to_playlist: FAILURE
remove_from_playlist: FAILURE
delete_song: SUCCESS
add_song: SUCCESS
get_by_plays: FAILURE
add_to_playlist: SUCCESS
get_num_songs: SUCCESS, 0
add_song: SUCCESS
add_to_playlist: SUCCESS
